mastracode/e2e/scenarios/index.ts

```
@@ -1,4 +1,5 @@
 import { activeSignalFollowupScenario } from './active-signal-followup.js';
+import { abortFollowupScenario } from './abort-followup.js';
```

ESLint

[error] `./abort-followup.js` import should occur before import of `./active-signal-followup.js`

(import/order)